src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1 +1 @@
-
+Microsoft.CodeAnalysis.Editing.DeclarationKind.Record = 29 -> Microsoft.CodeAnalysis.Editing.DeclarationKind
```

Comment:
For some reason the CI shows an error for the new enum member not being in PublicAPI, but I can't get the error locally in Visual Studio:

![image](https://user-images.githubusercontent.com/31348972/94978899-f657b400-051f-11eb-83ca-a345280f9592.png)

I added this manually, but the CI still complains. Not sure what should be the correct fix.